### DISCUSSION TITLE
Help: How do object methods in loop bodies pass parameters?

### DISCUSSION THREAD

**Discussion by Rookie259** [jxlsteam/jxls#252] [Q&A] [answered]
Help: How do object methods in loop bodies pass parameters?

At present, there are 5 loop bodies as dynamic headers. In the horizontal loop of the table body, for example, there is a loop object that wants to use the obj. getAnswer() method for ground penetrating parameters. How do I write a certain attribute of the 5 loop bodies?

Because the header is dynamic and the specific level is unknown, it needs to be obtained by the server based on the attributes of the lowest level.

Currently, jxls2 is used for exporting using Excel annotation templates.

![image](https://github.com/jxlsteam/jxls/assets/34497866/44c5c13b-b2c5-4249-ad25-9e74503447c6)

**Comment by SoltauFintel** [comment, marked as answer]
I don't know. I guess you have to prepare the whole matrix in Java memory and then send everything to the template.
